Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -16,6 +16,13 @@
     <DebugType>Full</DebugType>
   </PropertyGroup>
 
+  <!-- .NET Standard 1.4 doesn't have the Span<T> type, ValueTuple or System.Diagnostics.Contracts-->
```

Comment:
Indicate the required type for clarity.
Also, you missed a whitespace before the final `-->`.

```suggestion
  <!-- .NET Standard 1.4 doesn't have the Span<T> type, ValueTuple or the [Pure] attribute -->
```